src/app/business/profile/profileCard/profile-card.component.ts

```diff
@@ -147,20 +142,22 @@ export class ProfileCardComponent implements OnInit {
         }else{
             this.isAdministrator = false;
         }
+        this.chartDatas = {
+            labels: ['Unused Capacity', 'Used Capacity'],
+            datasets: [
+                {
+                    data: [this.data.totalFreeCapacity,this.data.totalCapacity-this.data.totalFreeCapacity],
+                    backgroundColor: [
+                        "rgba(224, 224, 224, .5)",
+                        "#438bd3"
+                    ]
+                }]
+        };
     }
 
     index;
     isHover;
 
-    getProfiles() {
-        this.allProfileNameForCheck = [];
-        this.ProfileService.getProfiles().subscribe((res) => {
-            let profiles = res.json();
-            profiles.forEach(item=>{
-                this.allProfileNameForCheck.push(item.name);
```

Comment:
do we need some UI intrinsic ordering here ? is the backend returned ordering deterministic ?

Reply:
I believe the backend returns ordered by created at timestamp but not sure if this is the case for all the APIs. 
This can be discussed with the API developers.

Reply:
please discuss and revert, with API dev. Thanks. This is key, across the system...for consistent UI experience

Reply:
I believe this is something that we will have to look at separately. Added an issue #190